WireAnalytics/Sources/WireAnalytics/Countly/Countly+Types.swift

```diff
@@ -16,8 +16,7 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Countly
+public import Countly
```

Comment:
So this ensures Countly is implicitly imported by users of `WireAnalytics`?

Reply:
curious about this too, why do we need to add public import here?